INSTALL.md

```diff
@@ -105,9 +105,9 @@ build command lines with V=1 (like kernel).
 CONFIGURATION
 =============
 
-The uftrace implements own version of configure script to save user
-preferences.  The config file (named `.config`) is created if not exist
-on build time with default options.  User can set custom installation
+The uftrace implements its own version of configure script to save user
+preferences.  The config file (named `.config`) is created, if non existent,
```

Comment:
I'm not sure if "[If non existent](https://www.google.com/search?q=%22if+non+existent%22&oq=%22if+non+existent%22&aqs=chrome..69i57.1046j0j1&sourceid=chrome&ie=UTF-8)" is more common than "[if not exist](https://www.google.com/search?q=%22if+not+exist%22&oq=%22if+not+exist%22&aqs=chrome..69i57.4296j0j1&sourceid=chrome&ie=UTF-8)".

Google search result show "if not exist" is much more common.

Reply:
> I'm not sure if "[If non existent](https://www.google.com/search?q=%22if+non+existent%22&oq=%22if+non+existent%22&aqs=chrome..69i57.1046j0j1&sourceid=chrome&ie=UTF-8)" is more common than "[if not exist](https://www.google.com/search?q=%22if+not+exist%22&oq=%22if+not+exist%22&aqs=chrome..69i57.4296j0j1&sourceid=chrome&ie=UTF-8)".
> 
> Google search result show "if not exist" is much more common.

`if not exist` is closer to code syntax (if !exists) than English (if [it does] not exist) so for documentation, I think it is better to change it.

If `non existent` feels awkward, how about `if missing` or `if not found` ?

Reply:
Then let's make it as a proper English sentence so I think `If it doesn't exist` is better.